package.json

```diff
@@ -25,6 +25,9 @@
     "jest-junit": "^12.0.0",
     "webpack-dev-server": "^3.11.0"
   },
+  "engines": {
+    "node": ">=14.17.6 <15"
+  },
```

Comment:
Alternatively, we could remove the `<15` requirement which would allow everyone to keep the newer node versions that they likely have installed.

Reply:
I think the `<15` is worth it, so that everyone upgrades together, even though it'll be annoying. The one-time pain will be worth it imho.